JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -124,7 +141,7 @@ int main(int argc,char**argv)
   TFile* ifile = new TFile(input.c_str(),"READ");
   if (!ifile->IsOpen()) { cout<<"Can't open "<<input<<endl; return 0; }
 
-  TFile* ofile = new TFile(output.c_str(),"UPDATE");
+  TFile* ofile = new TFile(output.c_str(),"RECREATE");
```

Comment:
Can you please make this an option? I personally like the UPDATE version. Please also uncomment lines 158-161 and then add something like that checks the TFile option to see if you should use those lines.